src/inversion_scripts/operators/operator_utilities.py

```diff
@@ -1,15 +1,43 @@
 import numpy as np
 import xarray as xr
 import pandas as pd
+import yaml
+from src.inversion_scripts.utils import (
+    check_is_OH_element,
+    check_is_BC_element
+)
+
+
+
 
 # common utilities for using different operators
+def read_all_geoschem(all_strdate, gc_cache, n_elements, config, build_jacobian=False, sensi_cache=None):
+    """
+    Call readgeoschem() for multiple dates in a loop.
+
+    Arguments
+        all_strdate    [list, str] : Multiple date strings
+        gc_cache       [str]       : Path to GEOS-Chem output data
+        build_jacobian [log]       : Are we trying to map GEOS-Chem sensitivities to TROPOMI observation space?
+        sensi_cache    [str]       : If build_jacobian=True, this is the path to the GEOS-Chem sensitivity data
```

Comment:
Do we need this variable anymore?

Reply:
Removed variable in https://github.com/geoschem/integrated_methane_inversion/pull/252/commits/d91d290e886919f3d89a3ca3f7729e607cdec341